src/components/datagrid/data_grid.test.tsx

```diff
@@ -1453,151 +1467,157 @@ Array [
           renderCellValue={({ rowIndex, columnId }) =>
             `${rowIndex}, ${columnId}`
           }
-          pagination={{
-            pageIndex: 0,
-            pageSize: 3,
-            pageSizeOptions: [3, 6, 10],
-            onChangePage: () => {},
-            onChangeItemsPerPage: () => {},
-          }}
+          pagination={pagination}
         />
       );
 
       let focusableCell = getFocusableCell(component);
+      // focus should begin at the first cell
```

Comment:
The chain of updates/expects grew to a point where it wasn't obvious what each block was doing/testing, so I added comments to each